components/blocks/FilesToCards.vue

```diff
@@ -59,36 +62,207 @@
+            placeholder="Sort by"
+            label="name"
+            track-by="name"
+            >\
+          </multiselect>
+        </span>
+        <span>
+          <button
+            class="button is-normal is-warning"
+            @click="ascending = !ascending"
+          >
+            <i v-if="ascending" class="mdi mdi-sort-ascending"></i>
+            <i v-else class="mdi mdi-sort-descending"></i>
+          </button>
+        </span>
+      </div>
+      <div
+        class="
+          container
+          card-container
+          is-flex
+          mt-6
+          is-justify-content-space-evenly is-flex-wrap-wrap
+        "
+      >
+        <DCard
+          v-for="(item, i) in sortedArray"
+          :key="'help-card-' + i"
+          class="mx-3 my-3 px-3"
+          variant="light"
+          accent="link"
+          width="medium"
+        >
+          <template #header>
+            <span v-if="item.group" class="radius-0 tag is-link has-text-light"
+              ><span class="icon"><i class="mdi mdi-account-multiple" /></span
+              ><abbr :title="item.group | expandAcronym">
+                {{ item.group }}
+              </abbr></span
+            >
+            <h2>{{ item.title }}</h2>
+            <div v-if="item.date">Updated: {{ item.date }}</div>
+            <span v-if="item.authors" class="small has-text-link"
+              >Authors: {{ item.authors.map((a) => a.name).join(', ') }}</span
+            >
+          </template>
+          <template #content>
+            {{ item.description }}
+          </template>
+          <template #footer>
+            <div v-if="item.links" class="link-group">
+              <a v-for="(link, type) in item.links" class="link-item d-button has-text-link" :href="link" :key="type">
+              <span>{{ type.toUpperCase() }}
+                <span class="icon ml-2">
+                  <i :class="['mdi', LINK_ICONS[type]]" />
+                  </span>
+                </span>
+              </a>
+            </div>
+          </template>
+        </DCard>
+      </div>
+    </div>
   </main>
 </template>
 
 <script>
 import DCard from '@/components/base/DCard.vue';
+import Multiselect from 'vue-multiselect';
 
 export default {
   components: {
     DCard,
+    Multiselect,
   },
   props: {
     data: {
       type: Array,
       required: true,
     },
   },
+  data: () => ({
+    ascending: true,
+    sortBy: [],
+    searchGroup: [],
+    sortOptions: [],
+    LINK_ICONS: {repository: "mdi-code-greater-than-or-equal", 
+              website: "mdi-link-variant", 
+              documentation: "mdi-book-open", 
+              publication: "mdi-newspaper",
+              doi: "mdi-book"}
+  }),
   computed: {
     filteredData() {
       return this.data.filter((d) => !d.hidden);
     },
+    sortedArray() {
+      let filtered = this.data;
+      if (this.searchGroup.length > 0) {
+        filtered = this.data.filter((card) => {
```

Comment:
should this go off of `this.filteredData` instead? or maybe have that `!hidden` check as part of this computed method?